docs/conf.py

```diff
@@ -63,6 +63,8 @@
 # directories to ignore when looking for source files.
 exclude_patterns.append('_templates')
 
+suppress_warnings += ['ref.doc']
```

Comment:
@SaraOgaz , turns out this is the magical incantation to make the Sphinx test pass. Thanks for paving the way to make this happen!